tools/prepare-machine.ps1

```diff
@@ -379,11 +379,7 @@ if ($Cleanup) {
     }
 
     if ($ForNetPerfTest) {
-        Setup-VcRuntime
-        Download-CoreNet-Deps
-        Download-Ebpf-Msi
-        Setup-TestSigning
-        Install-Certs
+        $ForTest = $true
```

Comment:
Perf tests require `Setup-VsTest` now? If they really do, perhaps we just get rid of `$ForNetPerfTest` if it's the same as `$ForTest`?

Reply:
I'm not sure why `ForNetPerfTest` was added in the first place with copy/paste of specific dependencies, but yeah, completely deleting it seems best.